packages/scripts/README.md

```diff
@@ -59,7 +59,19 @@ We commit to keeping the breaking changes minimal so you can upgrade `@wordpress
 
 ### `build`
 
-Transforms your code according the configuration provided so it’s ready for production and optimized for the best performance. The entry points for your project get detected by scanning all script fields in `block.json` files located in the `src` directory. The fallback entry point is `src/index.js` (other supported extensions: `.jsx`, `.ts`, and `.tsx`) in case there is no `block.json` file found. The output generated will be written to `build/index.js`. This script exits after producing a single build. For incremental builds, better suited for development, see the [start](#start) script.
+Transforms your code according the configuration provided so it’s ready for production and optimized for the best performance. The entry points for your project get detected by scanning all script fields in `block.json` files located in the `src` directory. The script fields in `block.json` should pass relative paths to `block.json` in the same folder.
+
+_Example:_
+
+```json
+{
+	"editorScript": "file:index.js",
+	"editorStyle": "file:editor.css",
+	"style": "file:style.css"
+}
+```
+
+The fallback entry point is `src/index.js` (other supported extensions: `.jsx`, `.ts`, and `.tsx`) in case there is no `block.json` file found. The output generated will be written to `build/index.js`. This script exits after producing a single build. For incremental builds, better suited for development, see the [start](#start) script.
```

Comment:
is the fallback of `src/index.js` equivalent to `file:index.js` or `file:src/index.js`?

... I think it's still a bit confusing!

Reply:
It depends on where your `block.json` is located. I plan to extend those docs with examples of the project structure:

- src
  - block.json
  - index.js

In this case, you put in `src/block.json` to reference `src/index.js`:
```json
"editorScript" : "file:index.js"
```

When that happens, `npm run build` + `register_block_type( __DIR__ . '/build' );` handles everything for you.

If you don't add `editorScript` entry or any other scripts in the JSON file, then `npm run build` will still process `src/index.js` as the only entry point, but you will have to register and enqueue the script yourself.